validator/specs/lib/validateTilesetSpec.js

```diff
@@ -77,3 +78,60 @@ describe('validateTileset', function() {
             }), done).toResolve();
     });
 });
+
+
+describe('validateBoundingVolume', function() {
+   var tileset = {
+       "asset": {
+           "version": "0.0"
+       },
+       "geometricError": 240,
+       "root": {
+           "boundingVolume": {
+               "region": [-1.3197209591796106, 0.6988424218, -1.3196390408203893, 0.6989055782, 0, 88]
+           },
+           "geometricError": 70,
+           "refine": "add",
+           "children": [
+               {
+                   "boundingVolume": {
+                       "region": [-1.3197209591796106, 0.6988424218, -1.31968, 0.698874, 0, 20]
+                   },
+                   "geometricError": 50,
+                   "content": {
+                       "url": "a.b3dm"
+                   },
+                   "children": [
+                       {
+                           "boundingVolume": {
+                               "region": [-1.3197209591796106, 0.6988424218, -1.31968, 0.698874, 0, 10]
+                           },
+                           "geometricError": 0,
+                           "content": {
+                               "url": "b.b3dm"
+                           }
+                       }
+                   ]
+               },
+               {
+                   "boundingVolume": {
+                       "region": [-1.31968, 0.6988424218, -1.3196390408203893, 0.698874, 0, 20]
+                   },
+                   "geometricError": 0,
+                   "content": {
+                       "url": "c.b3dm"
+                   }
+               }
+           ]
+       }
+   };
+
+    it('validates region inside region', function(done) {
+        expect(validateBoundingVolume(tileset)
+            .then(function(response) {
+                expect(response.result).toBe(true);
+                expect(response.message).toBe('Tileset is valid');
+            }), done).toResolve();
+    });
```

Comment:
Along with these test, add some failure cases like:
* Content region max height is taller than tile region
* Content region min height is smaller than tile region
* Individually test cases where the content region's west/east/south/north values exceed the tile region